### ISSUE TITLE
no pokestop visited

### ISSUE THREAD

**Issue by anonymouslulz22** [PokemonGoF/PokemonGo-Bot#1168]
### Expected Behavior
-m farm

### Actual Behavior
finds a pokestop starts walking but never reach the pokestop
at last it also shows no pokestop visited
### Steps to Reproduce

[15:28:23] Found fort f0178a7075a9445c980014b79f571f1a.16 at distance 0.05km
[15:28:23] Need to move closer to Pokestop
[                                        ] 0%
just shows this and then shows recycle item and then finds new pokestop and then again repeat it

### Other Information
OS: kali
Git Commit: (run 'git log -n 1 --pretty=format:"%H"' and paste it here)

**Comment by binarydepartment** [comment]
I can confirm this

**Comment by unequaled86** [comment]
probaly problem in step_walker

**Comment by BoGnY** [comment]
I confirm this.

> [12:17:09] Starting PokemonGo Bot....
> [12:17:09] Scanning area for objects....
> [12:17:09] Walking from (45.6668893, 12.2430437) to [45.6668893, 12.2430437] 0.00km
> [12:17:11] Starting to recycle items...
> [12:17:16] Finished.
> [12:17:16] Something rustles nearby!
> [12:17:17] A Wild Doduo appeared! [CP 193] [Potential 0.49]
> [12:17:17] IV [Stamina/Attack/Defense] = [8/13/1]
> [12:17:20] Using Pokeball (chance: 50.05%)... (87 left!)
> [12:17:21] Captured Doduo! [CP 193] [8/13/1]
> [12:17:26] Found fort d73f0df9e32d4824be76e678d99ab1d0.16 at distance 0.02km
> [12:17:26] Need to move closer to Pokestop
> [12:17:30] Scanning area for objects....
> [12:17:30] Walking from (45.66686601588999, 12.243045108533014) to [45.668689300000004, 12.2430437] 0.20km
> [12:17:34] Starting to recycle items...
> [12:17:39] Finished.

Maybe fixed after #1171 is commited

;)

**Comment by binarydepartment** [comment]
It's a duplicate of this: #1155 
You have to update your requirements, then it works again.

**Comment by anonymouslulz22** [comment]
[16:13:52] Found fort f0178a7075a9445c980014b79f571f1a.16 at distance 0.04km
[16:13:52] Need to move closer to Pokestop
[                                        ] 0%[16:13:59] Scanning area for objects....
[16:13:59] Walking from (40.77133480426469, -73.97368388875493) to [40.773161816544814, -73.97372463574011] 0.20km
[                                        ] 0%[16:14:20] Found fort f0178a7075a9445c980014b79f571f1a.16 at distance 0.04km
[16:14:20] Need to move closer to Pokestop

**Comment by anonymouslulz22** [comment]
still doing it
just finding and not moving

**Comment by unequaled86** [comment]
in move_to_fort.py
self._step_walker.step() returning none.
problem is here

**Comment by anonymouslulz22** [comment]
yeah
so whats the fix?
for it?
it should return something

**Comment by unequaled86** [comment]
in move_to_fort.py

find this
if not self._step_walker.step():
     return WorkerResult.RUNNING

change with this
step_walker = self._step_walker.step()
			if step_walker == True:
				return WorkerResult.SUCCESS
			elif step_walker == False:
				return WorkerResult.RUNNING

**Comment by vicch** [comment]
@unequaled86 Looks like it fixed the problem, thanks.

**Comment by BoGnY** [comment]
Thanks it work :)

**Comment by torkap** [comment]
@unequaled86 when I replace that (in pokemongo_bot/cell_workers/move_to_fort.worker.py, there is no move_to_fort.py anywhere? Latest dev) I get a KeyError:'returns' and cannot log in.

Did I edit the correct file?

**Comment by BoGnY** [comment]
yes you edit the correct file, but you must respect the line indeent..

**Comment by elfenars** [comment]
@unequaled86 doesn't really work for me, it always says that it's close to a pokestop but never actually collects it because it's too far away. 

This goes on and on, the bot actually moves, but it's not collecting any of the pokemons or the stops.

**Comment by BoGnY** [comment]
@unequaled86 I've just try old commits, 97658283f37db827af940f5d74146346ccfcdf06 commit work perfectly.. after that, pokestop don't spinned correctly..

**Comment by tstumm** [comment]
Please try with latest `dev`

**Comment by elfenars** [comment]
@tstumm it's working now

**Comment by DimaVIII** [comment]
@tstumm It's working now, Thanks! 
It it possible that the percentage bar "[==============                          ] 37%" is gone now? 

### Was:
[14:24:34] Need to move closer to Pokestop
[==============                          ] 37%[14:24:51] Found fort 40f2c3c220494f6f9730821e9572c415.16 at distance 0.02km

### IS:
[16:04:58] [x] Need to move closer to Pokestop
[16:05:08] [x] Found fort 1983253931184629aa9403c924c3428a.16 at distance 0.01km
[16:05:08] [x] Need to move closer to Pokestop

**Comment by tstumm** [comment]
It was accidentally made unfunctional.
